ViewController.swift

```diff
@@ -7,13 +7,100 @@
 
 import UIKit
 
-class ViewController: UIViewController {
-
+class ViewController: UICollectionViewController, UIImagePickerControllerDelegate, UINavigationControllerDelegate {
+    var people = [Person]()
```

Comment:
I think the best way to declare this array would be
```suggestion
    var people: [Person] = []
```

Here is a recent thread about it: https://redspace.slack.com/archives/C07QB769H/p1616420247003300.

Reply:
That's interesting: 
I'll change it!